docs/src/index.md

```diff
@@ -5,3 +5,8 @@
 [![Build Status](https://travis-ci.org/invenia/CodecLz4.jl.svg?branch=master)](https://travis-ci.org/invenia/CodecLz4.jl)
 [![Build Status](https://ci.appveyor.com/api/projects/status/github/invenia/CodecLz4.jl?svg=true)](https://ci.appveyor.com/project/invenia/codeclz4-jl)
 [![CodeCov](https://codecov.io/gh/invenia/CodecLz4.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/invenia/CodecLz4.jl)
+
+```@autodocs
+Modules = [CodecLz4]
+Pages = ["CodecLz4.jl", "lz4frame.jl", "stream_compression.jl"]
```

Comment:
If you're not using TranscodingStreams in Modules do you still need the Pages listed?

Reply:
Just checked, `[Pages]` is no longer needed, good call!